### PR TITLE
Team page gifs

### PR DESCRIPTION
**Pull request by rpietraszko**
## TO DO:
* [X] https://trello.com/c/VmdyslNy/13-mixin-bcg-images-refactor - `bcg-team-images` mixin is not needed right now, I've removed it and left the `bcg-company-images`
* [X] add new gifs, remove placeholder for Piotrek
* [X] make `team-member` as `img` instead of `div` and replace its source on hover
* [X] remove flipping module
* [ ] https://trello.com/c/ogntau14/35-team-page-picture-reverses-join-us-circle

---

Review:
* [ ] @MagdaMalinowska
* [ ] @aberracja
* [x] @kruszczynski
* [ ] @pjar
* [x] @AdamMankowski

### CODE REVIEW

File: source/partials/_team_join_us.html.slim [hunk 1]
```diff
@@ -0,0 +1,16 @@
+a.join-us-link(href='/join-our-crew')
+  figure.team-element class=element.type
+    .m-flipping
+      .flipping-container
+        .flipping-front(class="color-#{element.color}"
```
Comment (AdamMankowski) [inline source/partials/_team_join_us.html.slim]: We don't have any flipping anymore. Maybe it makes sense to change names ?

Comment (rpietraszko) [inline source/partials/_team_join_us.html.slim]: Good point, flipping is gone now

**Comment by AdamMankowski** [comment]
So much code deleted. Kudos!

**Comment by MagdaMalinowska** [comment]
Not technical feedback:
* Could you with @maretamara fix Marcin's gif?:) It looks like his head is growing. 
* Also Bartek S looks weird when you hover to cheerful photo and see much sadder gif:P 
* Tomek looks like in slow motion

**Comment by AdamMankowski** [comment]
@MagdaMalinowska it is bein taken care of.

**Comment by AdamMankowski** [comment]
I'm approving this PR but @rpietraszko please wait with merging it until i will provide better gifs

**Comment by AdamMankowski** [comment]
I've added the new gifs so for me it's ready to merge

**Pull request merged**